addons/point_of_sale/models/pos_order.py

```diff
@@ -152,6 +187,38 @@ def _process_order(self, pos_order):
                 'payment_name': _('return'),
                 'journal': cash_journal_id,
             })
+
+    @api.model
+    def _process_order(self, pos_order, draft = False, existing_order = False):
```

Comment:
you can put back all logic here instead of splitting in three functions; It'll make a better diff, and the functions are called only once and not overridden.